Playbooks/AS-Incident-Response-Approval-Email/README.md

```diff
@@ -0,0 +1,163 @@
+# AS-Incident-Response-Approval-Email
+
+Author: Accelerynt
+
+For any technical questions, please contact info@accelerynt.com  
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FAS-Incident-Response-Approval-Email%2Fazuredeploy.json)
+[![Deploy to Azure Gov](https://aka.ms/deploytoazuregovbutton)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FAS-Incident-Response-Approval-Email%2Fazuredeploy.json)       
+
+This playbook is intended to be run from a Microsoft Sentinel incident. It will facilitate incident response by sending an approval email to the manager(s) of the user(s) associated with the Microsoft Sentinel incident.
+
+![ApprovalEmail_Demo](Images/ApprovalEmail_Demo.png)
+ 
+                                                                                                                                
+#
+### Requirements
+                                                                                                                                     
+The following items are required under the template settings during deployment: 
+
+* An [App Registration](https://github.com/Azure/Azure-Sentinel/tree/master/Playbooks/AS-Incident-Response-Approval-Email#create-an-app-registration) for using the Microsoft Graph API
+* An [Azure Key Vault Secret](https://github.com/Azure/Azure-Sentinel/tree/master/Playbooks/AS-Incident-Response-Approval-Email#create-an-azure-key-vault-secret) containing your App Registration Secret 
+
+
+# 
+### Setup
+                                                                                                                                     
+#### Create an App Registration:
```

Comment:
Nicley explained steps for app registration and Azure Key vault setup. 
Can you numbered these steps for clarity wherever applicable?

Reply:
@rahul0216 I have added numbers to signify the ordering of steps

Reply:
Here's a screenshot of the testing that was done for this as well:
![image](https://user-images.githubusercontent.com/101294083/218598788-5e8720ad-e2cd-4c53-968a-aa3012637d9f.png)